django-stubs/forms/utils.pyi

```diff
@@ -13,26 +15,54 @@ _FilesT = MultiValueDict[str, UploadedFile]
 def pretty_name(name: str) -> str: ...
 def flatatt(attrs: Dict[str, Any]) -> SafeString: ...
 
-class ErrorDict(dict):
+class RenderableMixin:
+    def get_context(self) -> Dict[str, Any]: ...
+    def render(
+        self,
+        template_name: Optional[str] = ...,
+        context: Optional[Union[Context, Dict[str, Any]]] = ...,
+        renderer: Optional[BaseRenderer] = ...,
+    ) -> SafeString: ...
+    __str__ = render
+    __html__ = render
+
+class RenderableFormMixin(RenderableMixin):
+    def as_p(self) -> SafeString: ...
+    def as_table(self) -> SafeString: ...
+    def as_ul(self) -> SafeString: ...
+    def as_div(self) -> SafeString: ...
+
+class RenderableErrorMixin(RenderableMixin):
+    def as_json(self, escape_html: Optional[bool]) -> str: ...
+    def as_text(self) -> SafeString: ...
+    def as_ul(self) -> SafeString: ...
+
+class ErrorDict(dict, RenderableErrorMixin):
+    template_name: str = ...
+    template_name_text: str = ...
+    template_name_ul: str = ...
+    def __init__(
+        self,
+        data: Optional[Dict[str, Any]] = ...,
+        renderer: Optional[BaseRenderer] = ...,
+    ) -> None: ...
     def as_data(self) -> Dict[str, List[ValidationError]]: ...
     def get_json_data(self, escape_html: bool = ...) -> Dict[str, Any]: ...
-    def as_json(self, escape_html: bool = ...) -> str: ...
-    def as_ul(self) -> str: ...
-    def as_text(self) -> str: ...
 
-class ErrorList(UserList):
+class ErrorList(UserList, List, RenderableErrorMixin):
```

Comment:
I expect the tests to fail here, as this class now inherits from both `UserList` and `List`. 

https://github.com/django/django/blob/90d2f9f41671ef01c8e8e7b5648f95c9bf512aae/django/forms/utils.py#L138

🤷

Reply:
`List[Something]` might be good enough.

Reply:
Looks like that will do. Thank you!